tests/build.gradle

```diff
@@ -16,6 +16,8 @@ tasks.withType(Test) {
 }
 
 task testLean(type: Test) {
+    exclude '**/*DockerExampleContainer*'
+    exclude '**/*Java*'
```

Comment:
have no sniff tests for python, swift, and now java in travis is not overall goodness.
in #2070, there will be at least one test for each of these... can we add a comment here that explain we're skipping some tests, but all runtimes are still tested in lean?